test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/RetrieveTransactionResourceTests.Frame.cs

```diff
@@ -63,18 +63,6 @@ await foreach (Stream item in response)
         }
     }
 
-    [Theory]
-    [MemberData(nameof(GetUnsupportedAcceptHeadersForFrames))]
-    public async Task GivenUnsupportedAcceptHeaders_WhenRetrieveFrame_ThenServerShouldReturnNotAcceptable(bool singlePart, string mediaType, string transferSyntax)
-    {
```

Comment:
I think we need this

Reply:
Oops!

Reply:
@jovinson-ms Actually, it looks like this was removed in main too: https://github.com/microsoft/dicom-server/commit/6d397cc4d20363e6bc26aea1f99f99360ee4e625